protected/controllers/SiteController.php

```diff
@@ -29,7 +29,7 @@ public function filters() {
 	public function accessRules() {
         return array(
             array('allow',  // allow all users
-                'actions'=>array('index','error','contact','mapbrowse','team','about','advisory','faq','term','help','privacy', 'login', 'loginAffiliate', 'logout', 'revoke', 'feed' ),
+                // 'actions'=>array('index','error','contact','mapbrowse','team','about','advisory','faq','term','help','privacy', 'login', 'loginAffiliate', 'logout', 'revoke', 'feed' ),
```

Comment:
Because this line has been commented out, any user can access the `/site/admin` page without logging into their user account which is a big security issue - try it without logging into the test admin account with this [URL](http://gigadb.gigasciencejournal.com:9170/site/admin). I think this line needs to be uncommented and the new functions in this pull request added into the array.